content/go/placement_group.go.md

```diff
@@ -0,0 +1,300 @@
+		return
+	}
+
+	fmt.Println(string(jsonFormat))
+}
+
+```
+
+Delete Placement Group
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+)
+
+func main() {
+
+	username := "set me"
+	apikey := "set me"
+	
+	placementId := 12345
+
+	sess := session.New(username, apikey)
+
+	service := services.GetVirtualPlacementGroupService(sess)
+
+	receipt, err := service.Id(placementId).DeleteObject()
+	if err != nil {
+		fmt.Printf("%s\n", err)
+		return
+	}
+
+	jsonFormat, jsonErr := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr != nil {
+		fmt.Println(jsonErr)
+		return
+	}
+
+	fmt.Println(string(jsonFormat))
+}
+```
+
+Get Guests from a Placement Group
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+)
+
+func main() {
+
+	username := "set me"
+	apikey := "set me"
+
+	placementId := 12345
+
+	sess := session.New(username, apikey)
+
+	service := services.GetVirtualPlacementGroupService(sess)
+
+	receipt, err := service.Id(placementId).GetGuests()
+	if err != nil {
+		fmt.Printf("%s\n", err)
+		return
+	}
+
+	jsonFormat, jsonErr := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr != nil {
+		fmt.Println(jsonErr)
+		return
+	}
+
+	fmt.Println(string(jsonFormat))
+}
+```
+
+Get Placement Groups of the Account
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+)
+
+func main() {
+
+	username := "set me"
+	apikey := "set me"
+	
+	sess := session.New(username, apikey)
+	accountService := services.GetAccountService(sess)
+
+	receipt, err := accountService.GetPlacementGroups()
+	if err != nil {
+		fmt.Printf("%s\n", err)
+		return
+	}
+
+	jsonFormat, jsonErr := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr != nil {
+		fmt.Println(jsonErr)
+		return
+	}
+
+	fmt.Println(string(jsonFormat))
+}
+```
+
+Order a Virtual Guest into a specific Placement Group
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/softlayer/softlayer-go/datatypes"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+	"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+
+	username := "set me"
+	apikey := "set me"
+
+	sess := session.New(username, apikey)
+
+	quantity := 1
+	location := "448994"
+	packageId := 835
+	presetId := 215
+	hostname := "softlayer"
+	domain := "example.com"
+	placementId := 12345
+
+	virtualGuests := []datatypes.Virtual_Guest{
+		{
+			Hostname:         sl.String(hostname),
+			Domain:           sl.String(domain),
+			PlacementGroupId: sl.Int(placementId),
+		},
+	}
+
+	// Build a skeleton SoftLayer_Product_Item_Price objects. To get the list of valid
+	// prices for the package use the SoftLayer_Product_Package:getItems method
+	prices := []datatypes.Product_Item_Price{
```

Comment:
I don't like having hard-coded price-ids in examples. Ideally the prices should be looked up from a list of keyNames. https://softlayer.github.io/python/orderVyatta/ would be a good example of what I am talking about. Let me know if you have questions about that.